bin/setup

```diff
@@ -120,35 +120,28 @@ verify_bundled_gems (){
   } 2>&1
 }
 
-postgres_is_running (){
-  echo 'Checking that postgres is running'
+verify_yarn_packages (){
+  echo 'Updating Yarn packages...'
+
+  bundle exec yarn install 2>&1
 }
 
-verify_database_configuration (){
-  if [ -f config/database.yml ]
-  then
-    bundle exec rake db:create:all
+verify_database_migrations (){
+  echo 'Setting up database...'
+  migrate_existing_database || setup_database
+}
 
-    if [ $? -ne 0 ]
-    then
-      echo 'Unable to create the databases for you. Please ensure your database.yml is configured for your system and run rake db:create:all'
-      exit 0
-    fi
-  else
-    echo 'The config/database.yml file does not exist. Please create it.'
```

Comment:
I thought it's ok to stop accounting for a missing `config/database.yml` since the file's checked into the codebase.